app/views/security_group/new.html.haml

```diff
@@ -1,40 +1,44 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "securityGroupFormController"}
+%form#form_div{:name           => "angularForm",
+   "ng-controller" => "securityGroupFormController as vm",
+   "miq-form"      => true,
+   "model"         => "vm.securityGroupModel",
```

Comment:
I would suggest `"ng-show" => "vm.afterGet"` even here for the `new` form to be consistent with the `edit` form.
But for that you will need a change in your JS here:
https://github.com/gildub/manageiq-ui-classic/blob/af30bf92afc6dd5ff78125a80286f826f9822cad/app/assets/javascripts/controllers/security_group/security_group_form_controller.js#L17

Change `vm.afterGet = false;` to `vm.afterGet = true;`, to make sure that the form loads.